include/odp/api/spec/crypto.h

```diff
@@ -566,6 +566,16 @@ typedef struct odp_crypto_session_param_t {
 	 */
 	odp_bool_t auth_cipher_text;
 
+	/** Hash result location will not overlap authentication range
+	 *
+	 *  This flag can be set to indicate that the hash result location
+	 *  will not overlap authentication range. Setting this flag may
+	 *  improve performance.
+	 *
+	 *  Default value is false.
+	 */
+	odp_bool_t hash_result_not_in_auth_range;
```

Comment:
This is a bit complex parameter name.  E.g. hash_result_overlap would be shorter and without negation in the name.

How often result would overlap the range? If it's rare, we could select default value not to overlap - although it's backwards incompatible.

Reply:
Hash result field outside of the authenticated range is the more common case, AFAIK.

How about hash_result_in_auth_range? I prefer "in_auth_range" over "overlap" as it is more descriptive?

My logic in the negative form was that if the parameter is set, then it hints that the result field is not in the auth range and otherwise it does not hint anything (may or may not be in the range). But maybe it is better just to describe the parameter well enough that the naming does not matter.